Game.Engine/wwwroot/js/connection.ts

```diff
@@ -279,7 +279,8 @@ export class Connection {
         this.sendPing();
         this.onConnected();
 
-        if (this.reloading) window.location.reload();
+        // if (this.reloading) window.location.reload(); 
```

Comment:
This is not the source of the problem. this is a symptom of a websocket being unexpectedly disconnected. 

This reloading behavior is a failsafe that makes the client restart when the connection is dropped or the server is updated.

without it, when a connection drops, it will stay dropped. 

I think this is the symptom not the disease. might be worth throwing an exception in that case instead... might make it easier to identify why that code is running.

Reply:
Oh okay, will re-look at this, but might not get time until weekend